### ISSUE TITLE
:q!

### ISSUE THREAD

**Issue by acocalypso** [ITRS-Group/check_k8s#41]

**Comment by erikdsjostrom** [comment]
Hello @acocalypso, I (hopefully) implemented a quick fix for this which you can try out. If it works I can implement something proper and get it on master.

https://github.com/ITRS-Group/check_k8s/tree/dev/json-decode-issue

**Comment by acocalypso** [comment]
Hi thanks for the quickfix.

The json issue is gone so far. Anyway im still not able to use it but it seems to be an local issue.
Im getting: UNKNOWN - [Errno -3] Temporary failure in name resolution

I try to connect to an openshift environment via the api

**Comment by erikdsjostrom** [comment]
Yeah, the error occurred during the error handling of a faulty request, so what you're seeing now is most likely the underlying issue.

Looks to me like you're unable to reach your destination. I don't know your environment, so it's hard for me to say what could be wrong, but perhaps check your firewall and/or DNS settings.
